db/migrate/20250501014159_add_foreign_key_to_power_of_attorney_request_decisions.rb

```diff
@@ -0,0 +1,8 @@
+class AddForeignKeyToPowerOfAttorneyRequestDecisions < ActiveRecord::Migration[7.2]
+  def change
+    add_foreign_key :ar_power_of_attorney_request_decisions,
```

Comment:
POA requests have one POA request resolution. POA request decision is a subtype of POA request resolutions using [Rails's delegated type feature](https://guides.rubyonrails.org/association_basics.html#delegated-types).

Reply:
I removed the fk migrations and tests still pass.